src/app/shared/store/provider.state.ts

```
@@ -945,4 +967,127 @@ export class ProviderState {
+  getLanguageList({ patchState }: StateContext<ProviderStateModel>, {}: providerActions.GetLanguageList): Observable<LanguageListItem[]> {
+    patchState({ isLoading: true });
+    return this.languageListService
+      .getLanguageList()
+      .pipe(tap((languageList: LanguageListItem[]) => patchState({ languageList, isLoading: false })));
+  }
+
+  @Action(providerActions.CreateStudySubject)
+  createStudySubject(
+    { dispatch, patchState }: StateContext<ProviderStateModel>,
+    { payload }: providerActions.CreateStudySubject
+  ): Observable<SubjectModel | void> {
```

Biome (1.9.4)

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)